content/en/docs/concepts/overview/components.md

```diff
@@ -122,6 +122,12 @@ about containers in a central database, and provides a UI for browsing that data
 A [cluster-level logging](/docs/concepts/cluster-administration/logging/) mechanism is responsible for
 saving container logs to a central log store with search/browsing interface.
 
+### Network Plugins
+
+[Network plugins](/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins) are software
+components that implement the container network interface (CNI) specification. They are responsible for
+allocating IP addresses to pods and enabling them to communicate with each other within the cluster.
```

Comment:
Watch out for one implication:
>  Because these are providing cluster-level features, namespaced resources for addons belong within the `kube-system` namespace.

Actually, network plugins can (and some do) live outside `kube-system`.
Have a think about how we'll cover that specific aspect and not mislead readers.

Reply:
I'm really not sure how should I address this implication @sftim. The paragraph I added is just a very brief description of CNIs, where would I add the (potential) namespace confusion clarification? Any suggestions @reylejano, @tengqm?

Reply:
Some CNI pointers in open PR #39890

Could be useful in future PR.

* [Documentation](https://www.cni.dev/docs/)

* [Specification](https://www.cni.dev/docs/spec/)

* [Supported container runtimes](https://github.com/containernetworking/cni#container-runtimes)

* [Reference plugins](https://www.cni.dev/plugins/current/#reference-plugins)

* [3rd party plugins](https://github.com/containernetworking/cni#3rd-party-plugins)

* [Introduction to CNI](https://youtu.be/YjjrQiJOyME)

* [CNI deep dive](https://youtu.be/zChkx-AB5Xc)

* [Webinar: Kubernetes and Networks](https://www.youtube.com/watch?v=GgCA2USI5iQ)